src/commands.py

```diff
@@ -273,3 +274,14 @@ def argos_info(ep):
     "\n-- Excluded:", len(info_data["excluded"]),
     "\n-- UD created:", len(info_data["ud"]),
     "\n-- transactions:", len(info_data["tx"]))
+
+
+def id_pubkey_correspondence(ep, id_pubkey):
+    if check_public_key(id_pubkey):
+        print(id_pubkey, "public key correspond to identity:", get_uid_from_pubkey(ep, id_pubkey))
+    else:
+        pubkey = get_pubkey_from_id(ep, id_pubkey)
+        if pubkey == "No matching identity":
+            print (pubkey)
+        else:
+            print(id_pubkey, "identity correspond to public key:", pubkey)
```

Comment:
Feel free to use the .format() version, the goal is that it should be ready for internationalization (even if that may be a long path ahead :) ), whatever way is chosen. I like the %s version because I'm more used to it but the .format() one is perfect as well.